netlify.toml

```diff
@@ -5,31 +5,9 @@
 # Redirects for the missing API docs
 [[redirects]]
   from = "/platform-api/*"
-  to = "https://seqera-docs-api.netlify.app/platform-api/:splat"
+  to = "https://seqera-docs-api.netlify.app/platform-api/seqera-api:splat"
```

Comment:
Is this right? Combined with the link above to `to: "/platform-api/seqera-api",` I feel like this will end up redirecting to content at `https://seqera-docs-api.netlify.app/platform-api/seqera-api/seqera-api` (eg. 2 x `seqera-api`).

Reply:
Correct. `/seqera-api` only exists on the introduction page, but every other page is either `/platform-api/[operation]` or `/platform-api/[entity]-info`. So this redirect will cause a bunch of 404s @dana-seqera